packages/lmi/pyproject.toml

```diff
@@ -32,20 +32,21 @@ dependencies = [
     "typing-extensions; python_version <= '3.11'",  # for typing.override
 ]
 description = "A client to provide LLM responses for FutureHouse applications."
-dynamic = ["version"]
 license = {file = "LICENSE"}
-name = "lmi"
+name = "fhlmi"
 readme = "README.md"
 requires-python = ">=3.11"
+# dynamic = ["version"]
+version = "0.0.1"
```

Comment:
Was this just for prototyping?

These packages are coversioned, we should keep them the same